src/app/lib/pageStyles/index.jsx

```diff
@@ -0,0 +1,15 @@
+import styled, { css } from 'styled-components';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+
+export const PageMarginStyles = css`
```

Comment:
I think it'd be clearer for those building other pages to have this named as this:
```suggestion
export const indexMarginStyles = css`
```

Reply:
It looks to me like this is used in the same way that other styles in [`components/Grid`](https://github.com/bbc/simorgh/blob/912702ff266eaa8da56aa7831c5843213889dbbe/src/app/components/Grid/index.jsx#L9-L18). 
Can we move this to that place, so we co-locate these page-level margins?

Reply:
I agree with renaming it, but I don't think the `components/Grid` is the right place to put them since these are not related to Grid. 
I think the `pageStyles` folder is self-explanatory and it makes sense to keep them next to the components where they are used?

Reply:
You're right. I agree that in `components/Grid` isn't the most ideal. 
I think having a `components/PageLayout` where we define components like `IndexMain` and move [`PageContainer` component](https://github.com/bbc/simorgh/blob/33cca19131ec655c6ad9d53562403d0a313ab84c/src/app/lib/pageStyles/PageContainer.js) there would be good.

Reply:
That sounds better, shall we place the `indexMarginStyles` on its own file too? So they can be imported and used in the `IndexMain` and `IndexPageContainer` (just renamed it).